base/interactiveutil.jl

```diff
@@ -134,30 +134,60 @@ if Sys.isapple()
     end
     clipboard() = read(`pbpaste`, String)
 
-elseif Sys.islinux()
+elseif Sys.islinux() || Sys.KERNEL === :FreeBSD
     _clipboardcmd = nothing
+    const _clipboardcmds = if Sys.islinux()
+        Dict(
+            :copy => Dict(
+                :xsel  => `xsel --nodetach --input --clipboard`,
+                :xclip => `xclip -silent -in -selection clipboard`,
+            ),
+            :paste => Dict(
+                :xsel  => `xsel --nodetach --output --clipboard`,
+                :xclip => `xclip -quiet -out -selection clipboard`,
+            )
+        )
+    elseif Sys.KERNEL === :FreeBSD
+        Dict(
+            :copy => Dict(
+                :xsel  => `xsel -c`,
+                :xclip => `xclip -silent -in -selection clipboard`,
+            ),
+            :paste => Dict(
+                :xsel  => `xsel -p`,
+                :xclip => `xclip -quiet -out -selection clipboard`,
+            )
+        )
+    end
```

Comment:
Since the only difference is in the options passed to xsel, I think it might be cleaner as
```julia
xsel_copy  = Sys.islinux() ? `xsel --nodetach --input --clipboard`  : `xsel -c`
xsel_paste = Sys.islinux() ? `xsel --nodetach --output --clipboard` : `xsel -p`

_clipboardcmds = Dict(:copy  => Dict(:xsel  => xsel_copy,
                                     :xclip => `xclip -silent -in -selection clipboard`),
                      :paste => Dict(:xsel  => xsel_paste,
                                     :xclip => `xclip -quiet -out -selection clipboard`))
```